File: Modul 6/Soal 1/PRAK601-2310817210019-NaufalElyzar.c
#include <stdio.h>

int main() {
    int baris, kolom;
    
    scanf("%d %d", &baris, &kolom);
    
    int matriks[baris][kolom];
    
    for (int x = 0; x < baris; x++) {
        for (int y = 0; y < kolom; y++) {
            scanf("%d", &matriks[x][y]);
        }
    }
    
    for (int x = 0; x < baris; x++) {
        for (int y = 0; y < kolom; y++) {
            printf("%d ", matriks[x][y]);
        }
        printf("\n");
    }
    
    return 0;
}

File: Modul 6/Soal 1/PRAK601-2310817210019-NaufalElyzar.py
baris,kolom = map(int,(input().split()))
matriks = []
x = 0
dalam = list(map(int, input().split()))
print("\n")
for i in range(baris):          
    matriks.append(dalam[x : x + kolom])
    x = x + kolom
for baris in matriks:
    for dalam in baris:
        print(dalam, end = " ")
    print( )